SlowCheetah/Logging/ITransformationLogger.cs

```diff
@@ -6,7 +6,28 @@ namespace SlowCheetah
     using System;
 
     /// <summary>
-    /// Interface for using an internal logger in <see cref="XmlTransformer"/>
+    /// Importance of a message
+    /// </summary>
+    public enum LogMessageImportance
+    {
+        /// <summary>
+        /// High importace. Prioritize
+        /// </summary>
+        High,
```

Comment:
Can you assign values to these? Low = 0, Normal = 1, High = 2. This way comparisons can be made based on the value.